src/system/unix/system.c

```diff
@@ -15,6 +15,7 @@
 #include <stddef.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <time.h>
```

Comment:
could this be accidental?

Reply:
Actually compensates for `pthread` becoming optional in an included header, which previously transitively included `time`